osu.Game/Overlays/BeatmapSet/FeaturedArtistBeatmapPill.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Resources.Localisation.Web;
+
+namespace osu.Game.Overlays.BeatmapSet
+{
+    public class FeaturedArtistBeatmapPill : CompositeDrawable
+    {
+        public FeaturedArtistBeatmapPill()
+        {
+            AutoSizeAxes = Axes.Both;
+        }
+
+        [BackgroundDependencyLoader(true)]
+        private void load(OsuColour colours, OverlayColourProvider colourProvider)
+        {
+            InternalChild = new CircularContainer
+            {
+                Masking = true,
+                AutoSizeAxes = Axes.Both,
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = colourProvider?.Background5 ?? colours.Gray2,
+                    },
+                    new OsuSpriteText
+                    {
+                        Margin = new MarginPadding { Horizontal = 10f, Vertical = 2f },
+                        Text = BeatmapsetsStrings.FeaturedArtistBadgeLabel.ToUpper(),
+                        Font = OsuFont.GetFont(size: 10, weight: FontWeight.SemiBold),
+                        Colour = OverlayColourProvider.Blue.Colour1,
```

Comment:
This is a rather minor point but I recall from [conversations](https://github.com/ppy/osu/pull/13866#issuecomment-878412438) [in the past](https://github.com/ppy/osu/pull/13866#issuecomment-878930389) that referring to `OverlayColourProvider` members via these static properties is generally frowned upon, as a component should be either fully adherent to a parent colour scheme (therefore resolving the colour provider and using it exclusively), or use scheme-agnostic colours, i.e. `colours.Blue` in this case (it is the same colour as `Blue.Colour1`).

Reply:
The way would probably be to migrate all these colours over to `OsuColour` and remove all these statics from `OverlayColourProvider`. I can look at it if no one has any further objections to that, not sure if we want these statics to live in `OverlayColourProvider` anymore.

Reply:
Can be done in a separate PR, either way. This isn't the first usage so it's not too important to worry about it at this stage.